ui/perfherder/compare/CompareTable.jsx

```diff
@@ -172,7 +176,15 @@ export default class CompareTable extends React.Component {
         <thead>
           <tr className="subtest-header bg-lightgray">
             <th className="text-left">
-              <span>{testName}</span>
+              <span>
+                {testName}
+                {frameworkName === 'talos' && (
+                  <a href={url} style={{ color: '#23527c' }}>
+                    {' '}
```

Comment:
This space can be moved outside outside the `<a>`. Just before it, so we avoid having the space underlined on hover